packages/wrangler/src/index.ts

```diff
@@ -212,6 +212,7 @@ export function createCLIParser(argv: string[]) {
 			alias: "j",
 			describe: `Experimental: support wrangler.json`,
```

Comment:
Is it still experimental?

Reply:
Not after this PR, but the _flag_ is experimental (we'll be entirely removing it soon). The wording could be clarified